src/assert.js

```diff
@@ -14,6 +14,25 @@ class Assert {
 
 	// Assert helpers
 
+	// Documents a "step", which is a string value, in a test as a passing assertion
+	step( message ) {
+		let result = !!message;
+
+		message = message || "You must provide a message to assert.step";
+
+		this.test.steps.push( message );
+
+		return this.pushResult( {
+			result,
+			message
+		} );
+	}
+
+	// Verifies the steps in a test match a given array of string values
+	verifySteps( steps, message ) {
+		this.deepEqual( steps, this.test.steps, message );
```

Comment:
The first two arguments should be swapped for our standard (actual, expected [, message ]) signature.